src/main/java/org/jabref/model/entry/Month.java

```diff
@@ -88,33 +88,41 @@
         if (testString.length() > 3) {
             testString = testString.substring(0, 3);
         }
-        Optional<Month> month = Month.getMonthByShortName(testString);
```

Comment:
The indent is wrong here. I have no clue, what is going wrong on your side. Do you use tabs instead of spaces?

Reply:
I don't know what went wrong. I used the auto formatter of Eclipse, so it could be the tabs. Now it looks better after replacing the tabs with spaces by hand. I should definetly change the IDE if this is a common problem.